lib/joken/error.ex

```diff
@@ -42,6 +42,14 @@ defmodule Joken.Error do
     Joken.Config.default_claims/1.
     """
 
+
+  def message(%__MODULE__{reason: :algorithm_needs_key}),
+    do: """
+    A map was expected for the key parameter in the signer creation. 
+    This is mandatory for: #{inspect(Signer.map_key_algorithms())}
+    #{inspect(Signer.map_key_algorithms())}
```

Comment:
I left a non-multiline suggestion before... sorry for the mess.

Here it goes:

```suggestion
    This is mandatory for: #{inspect(Signer.map_key_algorithms())}.
```

Reply:
Ah, didn't see that